src/main/java/com/ardaslegends/repository/war/WarRepositoryImpl.java

```diff
@@ -0,0 +1,25 @@
+package com.ardaslegends.repository.war;
+
+import com.ardaslegends.domain.Faction;
+import com.ardaslegends.domain.war.War;
+import com.ardaslegends.domain.war.WarParticipant;
+import lombok.val;
+import org.springframework.data.jpa.repository.support.QuerydslRepositorySupport;
+
+import java.util.Set;
+
+public class WarRepositoryImpl extends QuerydslRepositorySupport implements WarRepositoryCustom{
+
+
+    public WarRepositoryImpl(){
+        super(War.class);
+
+    }
+
+    @Override
+    public War findWarByAggressorsAndDefenders(Set<WarParticipant> aggressors,Set<WarParticipant> defenders) {
+        //QWar war
+
+        return new War();
+    }
```

Comment:
Is there something missing here?

Reply:
Yeah Cemil was starting to do that, but we don't actually need it. However it would probably be better to use this with QueryDSL instead of the current `@Query` approach so we can have better exception handling

Reply:
yeeeah, you want to  do that now or just creae an issue for it and finish that one later?

Reply:
Best would be to do it now because it is associated with this PR. Would you mind doing it quickly so I can remove the wrappedServiceExection thing in the meanwhile?

Reply:
If this can wait for tomorrow then yes cause I'm going to sleep in 2 minutes.

Reply:
Yeah sure, I'll do the other thing in the meanwhile and you can review it tomorrow

Reply:
Poggers and good night then. Happy coding